shapely/tests/test_io.py

```diff
@@ -1260,9 +1260,13 @@ def test_to_geojson_exceptions():
     ],
 )
 def test_to_geojson_point_empty(geom):
-    # Pending GEOS ticket: https://trac.osgeo.org/geos/ticket/1139
-    with pytest.raises(ValueError):
-        assert shapely.to_geojson(geom)
+    if shapely.geos_version < (3, 10, 2):
+        # Pending GEOS ticket: https://trac.osgeo.org/geos/ticket/1139
+        with pytest.raises(ValueError):
+            assert shapely.to_geojson(geom)
+        return
+    # Else, just confirm that the operation succeed
+    shapely.to_geojson(geom)
```

Comment:
Could we test that it roundtrips correctly?

Reply:
Sure, just changed it.